examples/fuse_examples/multimodality/image_clinical/multimodality_image_clinical.ipynb

```diff
@@ -260,269 +204,31 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# explain about FuseMedML data pipeline\n",
     "from fuse_examples.multimodality.image_clinical.dataset import isic_2019_dataset\n",
-    "\n",
-    "train_dl, valid_dl = isic_2019_dataset(size=size, reset_cache=True, post_cache_processing_func=None)\n"
+    "train_dl, validation_dl = isic_2019_dataset(data_path=data_dir, cache_path=cache_dir, reset_cache=reset_cache, reset_split_file=reset_split_file)"
```

Comment:
can you add an explanation on the difference between isic_2019_dataset and ISIC classes?